src/transform/src/normalize_ops.rs

```diff
@@ -41,7 +41,7 @@ impl crate::Transform for NormalizeOps {
                 crate::canonicalization::TopKElision::action(expr);
                 // (b) Fuse various like-kinded operators. Might enable further canonicalization.
                 crate::fusion::Fusion::action(expr);
-                // (c) Fuse join trees (might lift in-between Filters).
+                // (c) Fuse join trees (might lift in-between MFPs).
```

Comment:
Used to be Filters, but then Join fusion was smartened to include more general MFPs.